src/components/dropdown-menu/DropdownMenu.tsx

```diff
@@ -69,33 +83,40 @@ function DropdownContextMenu<T>(props: Omit<DropdownMenuProps<T>, 'trigger'>) {
   const ref = useRef<HTMLDivElement>(null);
   useOnClickOutside(ref, closePopperElement);
 
+  const { style = {}, ...otherWrapperProps } = wrapperProps as ElementProps<E>;
   return (
-    <HandleMenuContextDiv onContextMenu={handleContextMenu}>
-      {isPopperElementOpen && (
-        <Portal>
-          <div ref={ref}>
-            <div
-              ref={setPopperElement}
-              style={styles.popper}
-              {...attributes.popper}
-            >
-              <Menu>
-                <MenuItems
-                  itemsStatic
-                  onSelect={(selected) => {
-                    closePopperElement();
-                    onSelect(selected);
-                  }}
-                  {...otherProps}
-                />
-              </Menu>
+    <Wrapper
+      style={{ ...(props?.as === 'div' && { display: 'contents' }), ...style }}
```

Comment:
I think you are breaking the behavior when `as` is undefined

It's up to the user to specify everything including the style if they chose to pass the `as` prop.

Reply:
Sorry it is not breaking because of the default value.

But I think when the user sets the as prop to `div`, the component should not assume the user want `display: 'contents'` on it

Reply:
if the user does not pass the `as` props, I assume he needs to have the old behavior

Reply:
I agree with Daniel. We should only set `display: 'contents'` if `props.as` is not defined.

Reply:
> Sorry it is not breaking because of the default value.
> 
> But I think when the user sets the as prop to `div`, the component should not assume the user want `display: 'contents'` on it

it depends on how we want the component to behave

Reply:
It's a question giving meaning and coherence to the props.
not specifying `as` = "I let the component decide the what to render in a way that generally works"
specifying `as` = "I take over control and I decide exactly what the wrapper element should be and the value of its props, including the style attribute"